extensions/adobe/experience/campaign/experienceevent.schema.json

```diff
@@ -207,14 +207,15 @@
           "minimum": 1,
           "maximum": 32767
         },
-        "https://ns.adobe.com/experience/campaign/link/tags": {
-          "title": "Tags",
-          "type": "array",
-          "description":
-            "A list of tags qualifying the link. The URL category is one of those tags.",
-          "items": {
-            "type": "string"
-          }
+        "https://ns.adobe.com/experience/campaign/link/label": {
+          "title": "URL Label",
+          "type": "string",
+          "description": "The user-friendly label clicking on which the link opens."
+        },
+        "https://ns.adobe.com/experience/campaign/link/category": {
```

Comment:
This would work better as `https://ns.adobe.com/experience/campaign/linkCategory`, see #277 and #284 for details.